src/styles/index.css

```diff
@@ -0,0 +1,256 @@
+.card {
+  height: 408px;
+  max-width: 200px;
+  transition-duration: 0.6s;
+}
+
+.card__img {
+  text-align: center;
+  margin-bottom: 35px;
+}
+
+.card__img img {
+  width: 160px;
+  height: 134px;
+}
+
+.card__title h2 {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.card__title p {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  font-style: normal;
+  line-height: 14px;
+  color: #616070;
+  margin-top: 4px;
+  margin-bottom: 16px;
+}
+
+.card__rating {
+  height: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 12px;
+}
+
+.card__review p {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.card__price span {
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__btn button {
+  border: 1px solid transparent;
+  font-family: Roboto, sans-serif;
+  border-radius: 5px;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  color: #fff;
+  background-color: #00acdc;
+  width: 100%;
+  height: 40px;
+  margin-top: 4px;
+}
+
+.card__btn button:hover {
+  border: 1px solid #00acdc;
+  background-color: #fff;
+  color: #00acdc;
+}
+
+.card:hover {
+  cursor: pointer;
+  transform: scale(1.2);
+  transition-duration: 0.3s;
+}
+
+.card:hover .card__title h2 {
+  color: #34568b;
+}
+
+.card:hover button {
+  background-color: #fff;
+  color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.nav-logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav-bar {
+  margin: 0;
+}
+
+.nav-items {
+  display: flex;
+  list-style: none;
+  height: 60px;
+  align-items: center;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  position: relative;
+  text-align: center;
+  line-height: 60px;
+  margin: 0 10px;
+}
+
+ul li:first-child {
+  margin-left: 0;
+}
+
+ul li:last-child {
+  margin-right: 0;
+}
+
+a {
+  display: inline-block;
+  text-decoration: none;
+  color: #060b35;
+  height: 100%;
+}
+
+a:hover {
+  color: #00acdc;
+}
+
+a:visited {
+  color: #000;
```

Comment:
The color for visited links is set to black (#000). Ensure that this color choice maintains sufficient contrast with the background for accessibility purposes. Consider using a color that aligns with your design while ensuring readability.